vehicle/calibration_adapter/include/calibration_adapter/calibration_adapter_node.hpp

```diff
@@ -30,7 +30,7 @@
 
 class CalibrationAdapterNode : public CalibrationAdapterNodeBase
 {
-  using Velocity = autoware_vehicle_msgs::msg::VelocityReport;
+  using VelocityReport = autoware_vehicle_msgs::msg::VelocityReport;
```

Comment:
At that time I intended to follow the manner of autoware_advc_interface where `VelocityReport` seems preferred, but 
I missed that it is of the same type with `Velocity`. So now the updated version has cancelled all the relevant changes to avoid unpredictable errors.

Reply:
At that time I intended to follow the manner of `autoware_advc_interface` where `VelocityReport` seems preferred, but I missed that it is of the same type with `Velocity`. So now the updated version has cancelled all the relevant changes to avoid unpredictable errors.